slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -22,9 +25,43 @@ using viam::common::v1::PoseInFrame;
 #define MAX_COLOR_VALUE 255
 const std::string strRGB = "/rgb";
 const std::string strDepth = "/depth";
+const auto HEADERTEMPLATE =
```

Comment:
[q] should this be defined within the viam::utils namespace? Or is it okay to leave as a global